frontend/app/src/py-handler.ts

```diff
@@ -331,7 +331,7 @@ export default class PyHandler {
     this.logToFile(
       `Starting packaged python subprocess: ${executable} ${args.join(' ')}`
     );
-    this.childProcess = execFile(executable, args);
+    this.childProcess = spawn(executable, args);
```

Comment:
This we discussed in the standup. Is it needed for this change or is it remnant of your work trying to fix https://github.com/rotki/rotki/issues/2808 ?

Reply:
From the documentation, it seems that `execFile` is more oriented towards running an application and waiting for completion to react on the result and spawn is more oriented towards running a process in parallel. In all the other places we also seem to use `spawn` instead of `execFile`. 

The main effect seems to be that `spawn` will write to the buffers (stdio/stderr) while running while `execFile` will only do after termination. 

Thus I kept it since it makes more sense as a use-case